DigitalOffice/Tests/UserServiceTests/Commands/UserCreateCommandTests.cs

```diff
@@ -40,7 +40,8 @@ public void ShouldThrowExceptionWhenUserDataIsInvalid()
                 MiddleName = "Example",
                 Email = "Example@gmail.com",
                 Status = "Example",
-                Password = "Example"
+                Password = "Example",
+                IsAdmin = false
             };
```

Comment:
Like I've stated above, I think it's not needed in requests.

CreateUserRequest is a model that the user sends. So he's automatically not an admin by default.
The only case I see this property is relevant is this: Admin wants to create other admins. 
I think this might be a security breach or something. Better to leave it to the developers

Reply:
Idk, maybe someday we'll be doing some secure endpoints with its checks or something. But now we have more general system, as I see it.